pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
```

Comment:
I think you want to add the git archive package here too.

Reply:
Ok, thanks! Done